src/herder/test/UpgradesTests.cpp

```diff
@@ -2319,3 +2319,33 @@ TEST_CASE("validate upgrade expiration logic", "[upgrades]")
         REQUIRE(upgrades.mBaseReserve);
     }
 }
+
+TEST_CASE("upgrade from cpp14 serialized data", "[upgrades]")
+{
+    std::string in = "{\n"
```

Comment:
please use a raw string litteral instead of escaping like this

Reply:
fixed

Reply:
joko300370 move and run